guide/getting-started/visual-language.md

```diff
@@ -203,6 +223,35 @@ Think about Bitcoin’s visual language as a scale that ranges from muted to hig
 
 In digital product design, there are times when a minimally expressive approach is what’s needed. For example, when a user makes a transaction, we want them to focus. At other times, illustrations and stronger visual elements are more appropriate, and this is where we can take advantage of the creative freedom Bitcoin affords.
 
+## The Lightning network
+
+The Lightning network is a layer 2 solution for scaling Bitcoin. It allows people to make almost instant payments with very small fees.
```

Comment:
We don't need to explain what LN is on every page, this is already covered in the Tech Primer and other pages.
```suggestion
```

Reply:
Added a brief entry in the Glossary which can be linked to when #479 is merged.

Reply:
Thanks, added glossary link